pkg/kubelet/container/helpers_test.go

```diff
@@ -231,11 +238,78 @@ func TestExpandVolumeMountsWithSubpath(t *testing.T) {
 			},
 			expectedSubPath:   "",
 			expectedMountPath: "/foo",
+			expectedOk:        true,
+		},
+		{
+			name: "subpaths with empty annotations",
```

Comment:
just one nit: Can you rename the testcase to be "empty environment variable" instead of "annotations"?  This isn't actually testing an annotation in the Kubernetes sense.

Reply:
Changed